core/config/project_settings.cpp

```diff
@@ -1207,6 +1207,8 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF("application/config/project_settings_override", "");
 	GLOBAL_DEF_BASIC("audio/buses/default_bus_layout", "res://default_bus_layout.tres");
 	custom_prop_info["audio/buses/default_bus_layout"] = PropertyInfo(Variant::STRING, "audio/buses/default_bus_layout", PROPERTY_HINT_FILE, "*.tres");
+	GLOBAL_DEF_RST("audio/general/3d_panning_strength", 2.0f);
```

Comment:
Property range hints should be added with `custom_prop_info[]` (as done above) to specify a valid range for the setting. I suggest allowing values between 0.0 and 4.0 and with a step of 0.01.

Reply:
I'm tweaking the ranges a bit. It's an advanced setting so I'm fine having it be a broad range with a fine adjustment step. Let me know if you're completely sure the ranges should be the same. The defaults are different so it would feel strange to me to have them adjustable over the same range.

Reply:
Is there a reason the 2D and 3D defaults are different? I find the 2D default pretty good, and I'd like the 3D default to be the same. The current 3D default feels way too strong to me.

Reply:
The 2D and 3D values control fundamentally different values. I'm exposing the tightness parameter in SPCAP, and then just a scalar for panning in 2D. They're not comparable values, so I don't know how to make them the same.

Reply:
If we want to normalize the default values for each in ProjectSettings to 1.0 or something we could just hardcode a scalar into the places where these settings are used. Also I think I mentioned in another PR but I'm travelling right now so I'm not in a good position to pick a good default. I don't think I have access to headphones for at least another few days.

Reply:
> If we want to normalize the default values for each in ProjectSettings to 1.0 or something we could just hardcode a scalar into the places where these settings are used.

Sounds good to me :slightly_smiling_face:

Reply:
Cool, made a change to bake defaults into the nodes instead of into the project settings. I think you're right; it looks a bit cleaner that way.